src/schematools/db.py

```diff
@@ -1,82 +0,0 @@
-"""Datbase storage of metadata from imported Amsterdam schema files."""
-import json
-
-from sqlalchemy import inspect
-from sqlalchemy.orm import sessionmaker
-
-from schematools import models
-from schematools.utils import toCamelCase
-
-
-def fetch_table_names(engine):
```

Comment:
Deze is nog wel in gebruik in een cli command, die moet als enige nog blijven.

Reply:
Ik had 'm geïnlined in `show_tablenames` zodat de hele module weg kan.